modules/ROOT/pages/sync-function-api.adoc

```diff
@@ -26,7 +26,7 @@ ifdef::show_edition[:page-edition: {release}]
 ifdef::prerelease[:page-status: {prerelease}]
 :page-role:
 :page-partial:
-:description: pass:q[Use Sync Functionss to implement effective data routing and access control in the cloud-to-edge synchronization of enterprise data.]
+:description: pass:q[Use Sync Functions to implement effective data routing and access control in the cloud-to-edge synchronization of enterprise data.]
```

Comment:
Why did we add the pass:q markup if we don't really have any markup that we need to pass through?

Reply:
It seems without the pass markup the square brackets are rendered. Not sure why they're there either 🤷🏼 I'll keep them for now